docusaurus.config.js

```diff
@@ -148,6 +148,44 @@ module.exports = {
         },
       },
     ],
+    [
+      // Zeebe REST API docs content
+      "@docusaurus/plugin-content-docs",
+      {
+        id: "api-zeebe-docs",
+        path: "api/zeebe",
+        routeBasePath: "api/zeebe",
+        sidebarPath: require.resolve("./api/zeebe/zeebe-sidebars.js"),
+        editUrl: "https://github.com/camunda/camunda-docs/edit/main/",
+        lastVersion: "current",
+        versions: {
+          current: {
+            label: "0.1",
```

Comment:
I think that when the 8.5 release happens, this should update to 1.0. I'll add acceptance criteria to #3250 so that I don't lose track of this.